packages/graphql/src/services/sessions.ts

```diff
@@ -413,3 +549,42 @@ export async function getSession({ id }: { id: string }, ctx: ContextWithUser) {
 
   return session
 }
+
+interface GetLeaderboardArgs {
+  sessionId: string
+}
+
+export async function getLeaderboard(
+  { sessionId }: GetLeaderboardArgs,
+  ctx: ContextWithUser
+) {
+  const top5 = await ctx.prisma.session
```

Comment:
simply return the top 5 for now, but the participant itself will also need to be displayed (and what if the participant is in the top 5)